src/tests/unit_tls.cpp

```diff
@@ -627,11 +695,6 @@ void TLS_Handshake_Test::go()
       if(server_handshake_completed == false && m_server->is_active())
          server_handshake_completed = true;
 
-      if(client->is_closed() || m_server->is_closed())
-         {
-         break;
-         }
```

Comment:
This early-exit needed to be removed so that the peer of an aborting client/server receives the alert (and we can check our expectation in `Callbacks::tls_alert()`).